indexer/src/main/kotlin/com/openlattice/indexing/BackgroundIndexedEntitiesDeletionService.kt

```diff
@@ -92,47 +98,65 @@ class BackgroundIndexedEntitiesDeletionService(
             val w = Stopwatch.createStarted()
             //We shuffle entity sets to make sure we have a chance to work share and index everything
             val lockedEntitySets = entitySets.values
+                    .map { EntitySetForDeletion(it.id, it.name, it.partitions) }
                     .shuffled()
-                    .filter { it.name != "OpenLattice Audit Entity Set" } //TODO: Clean out audit entity set from prod
                     .filter { tryLockEntitySet(it.id) }
-                    .map { IndexerEntitySetMetadata.fromEntitySet(it) }
 
-            val totalDeleted = lockedEntitySets
+            val lockedDeletedEntitySets = deletedEntitySets
+                    .map { (id, partitions) -> EntitySetForDeletion(id, "Deleted entity set [$id]", partitions) }
```

Comment:
What is this magic string `"Deleted entity set [$id]"` ?

Reply:
we don't store names for deleted entity sets.

Reply:
(this string is what will print in the logs instead of the name)